### DISCUSSION TITLE
HTTP healthcheck endpoint?

### DISCUSSION THREAD

**Discussion by llamafilm** [netbox-community/netbox#16018] [Q&A]
On Netbox 3.7.3 I was polling `/api/` for a basic healthcheck, to validate that Netbox is alive.  It returned HTTP 200 and a very small json response, about 500 Bytes.  After upgrading to version 4.0.0, this endpoint returns 403 error.  Is there any other endpoint meant for healthchecks that would return 200 success?

**Comment by jcollie** [comment]
The Docker container has switched to using `/login/` for health checks. It would be nice to have a dedicated endpoint for this though.
